bootstrap

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+WORK_DIR="$PWD"
+ROOT_DIR="$(dirname "$0")"
+
+test -n "$ROOT_DIR"
+cd "$ROOT_DIR"
+
+if ! which libtoolize >/dev/null 2>&1
+then
+	echo "libtoolize: command not found, falling back to glibtoolize" >&2
+	alias libtoolize=glibtoolize
+fi
+
```

Comment:
i think we should have `set -x` here, otherwise it will not be clear where the script fails, if it does